File: src/com/l2kt/gameserver/network/clientpackets/RequestPrivateStoreManageBuy.java
package com.l2kt.gameserver.network.clientpackets;

import com.l2kt.gameserver.model.actor.instance.Player;

public final class RequestPrivateStoreManageBuy extends L2GameClientPacket
{
	@Override
	protected void readImpl()
	{
	}
	
	@Override
	protected void runImpl()
	{
		final Player player = getClient().getActiveChar();
		if (player == null)
			return;
		
		player.tryOpenPrivateBuyStore();
	}
}

File: src/com/l2kt/gameserver/network/clientpackets/RequestPrivateStoreManageBuy.kt
package com.l2kt.gameserver.network.clientpackets

class RequestPrivateStoreManageBuy : L2GameClientPacket() {
    override fun readImpl() {}

    override fun runImpl() {
        val player = client.activeChar ?: return

        player.tryOpenPrivateBuyStore()
    }
}